docs/summit.md

```diff
@@ -7,13 +7,16 @@ layout: default
 
 Moby Summits allow community members to meet in person and share what they're doing with the various Moby projects. This page contains links to previous summits summaries, with slides and videos when available, as well as information and planning documents for the next one.
 
-### Moby Summit LA, September 2017
+### Moby Summit Copenhagen, October 2017
 
-Register for [Moby Summit at OSS LA](https://www.eventbrite.com/e/moby-summit-los-angeles-tickets-35930560273) September 14, 2017 in Los Angeles.
+The next Moby Summit will take place as part of [DockerCon EU 2017](https://europe-2017.dockercon.com).  
+Event registration is free as a DockerCon Add On, and you can [add it to your registration now](https://europe-2017.dockercon.com/moby-summit/) now.
+
+We look forward to seeing you in Copenhagen, Denmark!
 
-If there are topics you want to see covered in this summit, please send a pull request to modify [the agenda](/events/summit-agenda-sept-2017/).
 
 ### Previous Moby Summits recaps
 
+* [Moby Summit, September 2017](https://blog.mobyproject.org/moby-summit-los-angeles-recap-a41e6acf81f8)
 * [Moby Summit, June 2017](http://mobyproject.org/blog/2017/06/26/moby-summit-recap/)
```

Comment:
@amirmc do you mind changing this to `https://blog.mobyproject.org/moby-summit-june-2017-recap-b7f9c44334a6` ?